tests/platform_tests/daemon/test_pcied.py

```diff
@@ -46,6 +46,10 @@
 def setup(duthosts, rand_one_dut_hostname):
     duthost = duthosts[rand_one_dut_hostname]
     daemon_en_status = check_pmon_daemon_enable_status(duthost, daemon_name)
+    ## add delay for pcied ready
+    if "Wistron_sw_to3200k" in duthost.facts["hwsku"]:
+        time.sleep(60)
```

Comment:
this is too hw specific, we need generic logic that works for hardware skus.

Reply:
@lguohan  please check https://github.com/sonic-net/sonic-platform-daemons/blob/7c0a326e1dcd2be74a80f6d3a2d7c4af084c2035/sonic-pcied/scripts/pcied#L31  , if we perform this test after some condition like warm/cold reboot, it will need to wait 60 seconds for pcied ready.

Reply:
please comment if it's generic case that we remove for specific hwsku.

Reply:
@lguohan any comment? thanks.

Reply:
@WistronNetwork Is `Winstron_sw_to3200k` the only hwsku needs extra sleep? In the future, maybe there are some variations of Wistron devices need this extra sleep too? Is string `Winstron_sw_to3200k` is generic enough to cover other possible HWSKUs?

Reply:
@wangxin  please check  https://github.com/sonic-net/sonic-platform-daemons/blob/7c0a326e1dcd2be74a80f6d3a2d7c4af084c2035/sonic-pcied/scripts/pcied#L31 , it's common for all sonic devices that need a minute waiting for pcied ready if pcied get restarted. so this is NOT issue just for Winstron_sw_to3200k, we add HWSKU check just in case not to impact other vendors.

Reply:
If it is common for all SONiC devices, then the test code should not do extra wait just for Winstron HWSKU. The test code should do extra wait for all platforms. Agree?

Reply:
@WistronNetwork I don't understand why you need this delay at this setup function. This function just checks the reading pcied is enabled or not. I am not sure why you need the 60 seconds to check the daemon enabled state. 
PCIED_MAIN_THREAD_SLEEP_SECS = 60 is the time out value for checking the pcie device status periodically.

Reply:
@sujinmkang For some extreme case, we need this. If pmon docker restart and we perform this case at that time, we need to wait for the "first run" of the pcied thread. So it need to wait 60 seconds.

Reply:
@wangxin @sujinmkang do we agree this is common for all SONiC devices? if do we'll update the commit, thanks.

Reply:
@WistronNetwork If it is common for all SONiC devices, can you go ahead and update the commit? Thanks!

Reply:
@wangxin it's been updated, please review, thanks.